Tests/ScrollingPageControlTests.swift

```diff
@@ -0,0 +1,80 @@
+//
```

Comment:
Thanks for adding these tests! It's definitely good to have these defaults covered. If you've got time, and want to go above and beyond, implementing some snapshot tests like we have for `ActiveLabel` would be good as well.

Reply:
I would like to, but I'm still having trouble with Carthage and/or SnapshotTesting. Once I can get that running, then I'll make some snapshot tests.

Reply:
I'm no expert on Carthage, but I think I figured it out. After cloning the repo, we need to setup the dependencies brought in via Carthage. You normally do that using the `carthage bootstrap` command. In this case, it will still fail because we're using an older version of the Pointfree snapshot testing library. Run the `carthage update` command to have Carthage update its dependencies (which also seems to cover what `carthage bootstrap` would normally do). After doing that, I was able to get the unit tests to compile and run (note that you'll need to run on iPhone X simulator).

Reply:
👍 that worked, snapshots coming soon...